tests/conf.js

```diff
@@ -18,8 +18,7 @@ const baseConfig = {
   seleniumAddress: 'http://localhost:4444/wd/hub',
   //Enabling specs as they are working.
   suites: {
-    // e2e:'e2e/**/*.js',
-    e2e: ['**/login.spec.js'],
+    e2e: ['**/login.spec.js','e2e/users/**/*.js'],
```

Comment:
In an attempt to make this less of a merge headache. I would push to the new line and add the trailing comma.

```
    e2e: [
      '**/login.spec.js',
     'e2e/users/**/*.js',
    ],